build-logic/buildquality/src/main/kotlin/gradlebuild/testcleanup/TestFilesCleanupService.kt

```diff
@@ -36,38 +42,40 @@ import java.util.stream.Collectors
 import java.util.zip.ZipEntry
 import java.util.zip.ZipOutputStream
 import javax.inject.Inject
-import kotlin.collections.Collection
-import kotlin.collections.List
-import kotlin.collections.Map
-import kotlin.collections.MutableList
-import kotlin.collections.MutableMap
-import kotlin.collections.any
-import kotlin.collections.associateWith
 import kotlin.collections.component1
 import kotlin.collections.component2
-import kotlin.collections.contains
-import kotlin.collections.emptyList
-import kotlin.collections.emptyMap
-import kotlin.collections.filter
-import kotlin.collections.flatMap
-import kotlin.collections.forEach
-import kotlin.collections.getOrDefault
-import kotlin.collections.getValue
-import kotlin.collections.isNotEmpty
-import kotlin.collections.joinToString
-import kotlin.collections.mutableListOf
-import kotlin.collections.mutableMapOf
-import kotlin.collections.plus
 import kotlin.collections.set
-import kotlin.collections.toTypedArray
 
 
 typealias LeftoverFiles = Map<File, List<String>>
 
 
 abstract class TestFilesCleanupService @Inject constructor(
     private val fileSystemOperations: FileSystemOperations
-) : BuildService<TestFileCleanUpRootExtension>, AutoCloseable, OperationCompletionListener {
+) : BuildService<TestFilesCleanupService.Params>, AutoCloseable, OperationCompletionListener {
+
+    interface Params : BuildServiceParameters, TestFilesCleanupBuildServiceRootExtension {
```

Comment:
I'd prefer adding the one field from the root project extension here instead of extending it.

Reply:
Done.